apps/web/Dockerfile

```
@@ -0,0 +1,48 @@
+RUN pnpm --filter @brx/db db:generate
+
+# Build the application
+RUN pnpm --filter @brx/web build
+
+# Production stage
+FROM base AS runner
+WORKDIR /app
+ENV NODE_ENV=production
+
+RUN addgroup -g 1001 -S nodejs
+RUN adduser -S nextjs -u 1001
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)